packages/core/integration-tests/test/kotlin.js

```diff
@@ -14,7 +14,7 @@ describe.skip('kotlin', function() {
   it('should produce a basic kotlin bundle', async function() {
     let b = await bundle(__dirname + '/integration/kotlin/index.js');
 
-    await assertBundleTree(b, {
+    await assertBundles(b, {
```

Comment:
test fails here with `TypeError: expectedBundles is not iterable`.

Reply:
the second argument has to be an array, e.g.:
```js
    assertBundles(b, [
      {
        type: 'js',
        assets: ['index.js']
      }
    ]);
```